ts/test/mock_transpiler.bzl

```diff
@@ -53,7 +53,7 @@ def mock(name, srcs, source_map = False, **kwargs):
         srcs = srcs,
         # Calculate pre-declared outputs so they can be referenced as targets.
         # This is an optional transpiler feature aligning with the default tsc transpiler.
-        js_outs = lib.calculate_js_outs(srcs, ".", ".", False, False, False, False),
-        map_outs = lib.calculate_map_outs(srcs, ".", ".", True, False, False) if source_map else [],
+        js_outs = lib.calculate_js_outs(srcs, ".", ".", False, False, False, False, False),
```

Comment:
these are getting gross, I think we should consider a post-factoring where we pass a struct to these helpers instead. (we could even pass the entire `ctx.attr` but feels violating law of demeter to me)